### PR TITLE
:crab: uwucounter

### PR DESCRIPTION
**Pull request by sourTaste000**
rip

### CODE REVIEW

**Comment by sourTaste000** [comment]
removed uwucounter

**Comment by iBuyMountainDew** [comment]
yes

File: package.json [hunk 1]
```diff
@@ -17,11 +17,11 @@
     "cat-facts": "^1.0.2",
     "child-process": "1.0.2",
     "discord.js": "discordjs/discord.js",
-    "enmap": "^5.4.1",
     "graceful-fs": "^4.2.4",
     "ms": "^2.1.2",
     "node-fetch": "^2.6.0",
     "opusscript": "^0.0.7",
+    "wikijs": "^6.0.1",
```
Comment (iBuyMountainDew) [inline package.json:24]: Why?

Comment (sourTaste000) [reply package.json:24]: oh this is for wiki

Comment (sourTaste000) [reply package.json:24]: i was going to add a wiki command

Comment (dominikaaaa) [reply package.json:24]: we already said no to that in the proper issue tho

**Comment by sourTaste000** [comment]
here done

**Comment by iBuyMountainDew** [comment]
Je approve. Counter is gone :crab:

**Comment by sourTaste000** [comment]
:crab:

**Comment by humboldt123** [comment]
non!

**Pull request merged**